<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>458</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>519</width>
        <height>436</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="7">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btn_previous">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="buttons.qrc">
             <normaloff>:/buttons/skip-previous.png</normaloff>:/buttons/skip-previous.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_pause">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="buttons.qrc">
             <normaloff>:/buttons/pause.png</normaloff>:/buttons/pause.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_play">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="buttons.qrc">
             <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_stop">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="buttons.qrc">
             <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_next">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="buttons.qrc">
             <normaloff>:/buttons/skip-next.png</normaloff>:/buttons/skip-next.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDial" name="pdiaVolume"/>
         </item>
        </layout>
       </item>
       <item row="2" column="7">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="m_Current">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="m_psldPosition">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_Remain">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="currentTrack">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="7">
        <widget class="QTableView" name="playlistView"/>
       </item>
       <item row="0" column="7">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Добавить треки</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="QToolButton" name="Lastplaylist">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="buttons.qrc">
                 <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QToolButton" name="Music">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="buttons.qrc">
                 <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QToolButton" name="btn_add">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="buttons.qrc">
                 <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Выбрать из Файлов</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Последн. сохр. Плейлист</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Этот Компьютер-&gt;Музыка</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Шалаш Анна гр.150504 - Аудиоплеер</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="0">
              <widget class="QToolButton" name="Clear">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="buttons.qrc">
                 <normaloff>:/buttons/Delete.png</normaloff>:/buttons/Delete.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QToolButton" name="Save">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="buttons.qrc">
                 <normaloff>:/buttons/Save.png</normaloff>:/buttons/Save.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Сохранить Текущий Плейлист</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Очистить Текущий Плейлист</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
